client/components/social-icons/google.jsx

```diff
@@ -19,13 +20,14 @@ export default class GoogleIcon extends PureComponent {
 	};
 
 	render() {
-		const props = omit( this.props, [ 'isDisabled' ] );
+		const props = omit( this.props, [ 'isDisabled', 'isLoading' ] );
 
 		return (
 			<svg
 				className={ classNames( 'social-icons social-icons__google', {
 					'social-icons--enabled': ! this.props.isDisabled,
 					'social-icons--disabled': !! this.props.isDisabled,
+					placeholder: this.props.isLoading,
```

Comment:
A better class name would be `social-icons--placeholder` because that better adheres to the BEM rules and avoids conflict with other usages of the very generic `placeholder` class name.